.ansible-lint-ignore

```diff
@@ -0,0 +1,83 @@
+# This file contains ignores rule violations for ansible-lint
```

Comment:
I think you can update the ansible-lint config to skip linting the roles/servicetelemetry/ directoty

Same for the build/run-stf-ci/ directory., although I think that the effort to update those is safer.

Reply:
Indeed, we can work on fixing the linting issues or configuring ansible-lint in a follow up patch.